webapp-django/crashstats/signature/static/signature/js/signature_tab_summary.js

```diff
@@ -18,3 +18,14 @@ SignatureReport.SummaryTab = function(tabName) {
 };
 
 SignatureReport.SummaryTab.prototype = SignatureReport.inherit(SignatureReport.Tab.prototype);
+
+SignatureReport.SummaryTab.prototype.onAjaxSuccess = function (contentElement, data) {
+    SignatureReport.Tab.prototype.onAjaxSuccess.apply(this, arguments);
+
+    $(contentElement).on('click', '.toggle-visible', function (e) {
```

Comment:
Why can't this just be `$(contentElement).on('click', '.panel header', function(e) {`??